src/geode/georepresentation/msh_input.cpp

```diff
@@ -111,10 +113,10 @@ namespace
     public:
```

Comment:
for struct hash< GmshElementID > you can use other hashes than string like `^ std::hash< geode::index_t >()( gmsh_id.id );`

Reply:
Done